testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -345,17 +345,17 @@ def test_verbose_progressbar(u, capsys):
 def test_verbose_progressbar_run(u, capsys):
     FrameAnalysis(u.trajectory).run(verbose=True)
     _, err = capsys.readouterr()
-    expected = u'100%|██████████| 98/98 [00:00<00:00, 8799.49it/s]'
+    expected = u'100%|██████████'
     actual = err.strip().split('\r')[-1]
-    assert actual[:24] == expected[:24]
+    assert expected == actual[:15]
```

Comment:
Why did you switch actual and expected?

Doesn't really matter except for consistency throughout because in [testsuite/MDAnalysisTests/lib/test_log.py](https://github.com/MDAnalysis/mdanalysis/pull/4732/files#diff-3f7a25a1baa99374633c43432096cd1139a273ab31a5d45d05843801aca8dcff) it's still actual == expected.

Reply:
I had originally switched it to a `expected in actual` check, but decided to revert back to an equality check.

Reply:
Reverted the order in the latest commit.